client_with_ssl.py

```diff
@@ -11,6 +11,61 @@
 See: https://docs.hazelcast.cloud/docs/python-client
 """
 
+
```

Comment:
Same comments in the other file applies here as well

Reply:
Fixed